admin-dev/themes/new-theme/js/pages/order/view/order-product-add.js

```diff
@@ -180,4 +182,29 @@ export default class OrderProductAdd {
       }
     });
   }
+
+  handleAddProductWithConfirmationModal(event) {
+    const invoiceId = parseInt(this.invoiceSelect.val(), 10);
+    const orderId = $(event.currentTarget).data('orderId');
+
+    if (invoiceId === 0) {
+      const modalTranslations = this.confirmNewInvoiceModalTranslations.data('translations');
+
+      const modal = new ConfirmModal({
+        id: 'modal-test',
```

Comment:
Maybe you should use an id more related to this context like `confirm_new_invoice_modal` ?
@NeOMakinG I never remember if it's better to use `-` or `_` in ids/classes?

Reply:
this one was indeed forgotten to modify :smile:

Reply:
I think we've decided to use `-` in classs and id.

Reply:
never use underscores, we follow bootstrap rules on selectors, prefer using dash :)